yarn-project/pxe/src/contract_function_simulator/pxe_oracle_interface.ts

```diff
@@ -336,69 +338,14 @@ export class PXEOracleInterface implements ExecutionDataProvider {
     }));
   }
 
+  // Note: I think syncSenderTaggingIndexes would be a better name as it better communicates the purpose of
+  // the function - we care about finding the indexes but we don't care about the log contents.
+  // TODO(benesjan): Rename the function to syncSenderTaggingIndexes if reviewer agrees.
   public async syncTaggedLogsAsSender(
```

Comment:
The whole algorithm has been replaced so it doesn't make sense to look at the deleted code. The new implementation is in the `syncSenderTaggingIndexes` function. Does the reviewer agree with the new name? I think the new name is better because the whole point of this is not to sync the logs (in a sense that we don't care about the log contents) but to sync the tagging indexes.

Anyway I think it would make sense to not rename the function here but to instead just go ahead and drop the whole PXEOracleInterface class along with the ExecutionDataProvider interface.